File: Day 25/Aayushi-Mittal.py
# To merge 2 array into a third array.

arr1 = []
arr2 = []
arr = []
n=int(input())
for i in range(0,n):
    arr1.append(int(input()))
m=int(input())
for i in range(0,m):
    arr2.append(int(input()))
    """
for i in range(0,n):
    arr.append(arr1[i])
for i in range(0,m):
    arr.append(arr2[i])
    """
arr = arr1 + arr2
print("Mergerd array arr[", m+n, "] : ")
for i in range(0,n+m):
    print(arr[i], end=" ")

File: Day 25/Aayushi-Mittal.cpp
//To merge 2 array into a third array.
#include<iostream>
using namespace std;
int main()
{
  int i, n=4, m=4, arr1[n], arr2[m], arr[m+n];
  cin>>n;
  for(int i=0; i<n; i++)
    cin>>arr1[i];
  cin>>m;
  for(int i=0; i<m; i++)
    cin>>arr2[i];
  for(int i=0; i<n; i++)
    arr[i]=arr1[i];
  for(int i=0; i<m; i++)
    arr[n+i]=arr2[i];
  cout<<"Mergerd array arr["<<m+n<<"] : ";
  for(int i=0; i<n+m; i++)
    cout<<arr[i]<<" ";
}
